config/300-trigger.yaml

```diff
@@ -77,27 +77,29 @@ spec:
                   additionalProperties:
                     type: string
             subscriber:
-              type: object
-              properties:
-                dnsName:
-                  type: string
-                  minLength: 1
-                uri:
-                  type: string
-                  minLength: 1
-                ref:
+              anyOf:
+                - uri:
```

Comment:
should be properties ?

Reply:
Is anyOf better since the logic is at least one here?

Reply:
the `anyOf` here has the same effect as a single object, since the anyOf's options don't overlap.